cmd/assetsvc/postgresql_utils.go

```diff
@@ -92,11 +103,28 @@ func (m *postgresAssetManager) getPaginatedChartList(namespace, repo string, pag
 	return charts, 1, nil
 }
 
-func (m *postgresAssetManager) getChart(namespace, chartID string) (models.Chart, error) {
+func (m *postgresAssetManager) getChart(namespace, chartIDUnescaped string) (models.Chart, error) {
+	return m.getChartWithFallback(namespace, chartIDUnescaped, enableFallbackQueryMode)
+}
+
+func (m *postgresAssetManager) getChartWithFallback(namespace, chartIDUnescaped string, withFallback bool) (models.Chart, error) {
```

Comment:
this chartIDUnescaped gets me confused. As far as I know this would be always escaped, no? The thing is that sometimes this chartID would be incomplete. So valid values for this function would be:
 1.  `bitnami/wordpress` for the normal bitnami wordpress
 2. `jfrog/bitnami%2Fwordpress` when installing the chart but from the jfrog repo
 3. `jfrog/wordpress` when upgrading the chart, when we don't have the `bitnami` info

am I wrong?

Reply:
Yes, totally right! We may have these three possible values. The `...Unescaped` var name is very confusing now. It made sense when dealing with encoding/decoding, but since it is directly stored as-is, I think we can rename them back.